config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/tool/tools_eu.properties

```diff
@@ -255,3 +255,6 @@ sakai.adminsiteperms.description=Administratzaileentzako tresna bat baimenak kud
 
 sakai.gradebookng.title=Ebaluazio liburua
 sakai.gradebookng.description=Gunean dauden ebaluatzeko itemak kudeatzeko tresna da.
+
+bulk.user.membership.title = Erabiltzaileen Kidetza-edizioa
```

Comment:
@pablosanroman would you mind providing a preferred name for this tool?

Reply:
Hi @mpellicer 
It is our suggestion, what do you think?:

> Bulk enrollment
> Altas masivas
> Alta masiboak

Reply:
Well, it's not just add operations, it also removes, activates and deactivates.

Reply:
Well, the EU name of the tool is totally up to you, if you like 'Alta masiboak' it's going to be 'Alta masiboak'

Reply:
Yes, @mpellicer , you are right. Let's see this other option:

> Bulk account management
> Gestión masiva de cuentas
> Kontuen kudeaketa masiboa

Reply:
We're not really managing accounts, we're managing memberships, I think it's confusing.

Gestión masiva de membresías which is Bulk Membership Management?

Reply:
Well....Bulk User Membership is accurate?

Reply:
Fine!, in my opinion..
and ES, perhaps in short way. "Gestión de pertenencia",  but ...for you the last word. :) 
and EU: "Kudeatu partaidetza"